docs/analytics_engine.rst

```diff
@@ -330,38 +326,35 @@ To run your own algorithms, you may trigger it in place where you defined it.
 .. code:: python
 
     import graphscope
+    from graphscope.dataset import load_p2p_network
 
-    sess = graphscope.session()
-    g = sess.g()
+    g = load_p2p_network()
 
     # load my algorithm
     my_app = SSSP_Pregel()
 
     # run my algorithm over a graph and get the result.
     # Here the `src` is correspondent to the `context.get_config(b"src")`
-    ret = my_app(g, src="0")
+    ret = my_app(g, src="6")
 
 
 After developing and testing, you may want to save it for the future use.
 
 .. code:: python
 
-    SSSP_Pregel.to_gar("file:///var/graphscope/udf/my_sssp_pregel.gar")
-
+    SSSP_Pregel.to_gar("/tmp/my_sssp_pregel.gar")
 
 Later, you can load your own algorithm from the gar package.
 
 .. code:: python
 
-    import graphscope
-
-    g = graphscope.g()
+    from graphscope.framework.app import load_app
 
     # load my algorithm from a gar package
-    my_app = load_app('file:///var/graphscope/udf/my_sssp_pregel.gar')
+    my_app = load_app("/tmp/my_sssp_pregel.gar")
 
     # run my algorithm over a graph and get the result.
-    ret = my_app(g, src="0")
+    ret = my_app(g, src="6")
```

Comment:
ditto. Have you checked the result's correctness?

Reply:
Check result in [UDF CI Test](https://github.com/alibaba/GraphScope/blob/main/python/graphscope/tests/unittest/test_udf_app.py#L1009)